### PR TITLE
Fix home page CSS

### PR DESCRIPTION
**Pull request by ektravel**
This PR:

- Fixes the issue with the grid
- Changes `<code>` background color
- Fixes navbar
- Fixes banner
- Changes footer text color

### CODE REVIEW

**Review by vtlim — APPROVED** [review]
Reviewed the following:
- [x] widget column really wide
- [x] Banner not matching (the location of the Druid icon)
- [x] CSS for inline code font isn’t monospace and darker grey background
- [x] Hoverbar in the top nav

File: src/pages/druid-powered.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ title: Powered by Apache Druid
   </div>
 </div>
 
+
 :::tip
 
-<a href="https://github.com/apache/druid-website-src/blob/master/src/pages/druid-powered.md"><span class="fa fa-plus"></span> Add Your Company</a>
+<a href="https://github.com/apache/druid-website-src/blob/master/druid-powered.md">
```
Comment (317brian) [inline src/pages/druid-powered.md:14]:
```suggestion
<a href="https://github.com/apache/druid-website-src/blob/master/src/pages/druid-powered.md">
```

Comment (317brian) [reply src/pages/druid-powered.md:14]: Looks like b/c of your additions, it's trying to override the URL fix Victoria made, so we can fix it manually here.

**Review by 317brian — APPROVED** [review]
The changes LGTM, especially the in line code fix. Great job @ektravel !

Just one minor suggestion re: the link on the druid-powered page

**Pull request merged**